aikau/src/main/resources/alfresco/core/CoreXhr.js

```diff
@@ -427,12 +428,15 @@ define(["dojo/_base/declare",
 
          // Build up the Accept-Language header value
          var languages = navigator.languages;
-         if (languages) {
+         if (languages) 
+         {
             languages = array.map(languages, function(nextLang) {
                return nextLang.toLowerCase();
             }).join(", ");
-         } else {
-            languages = (navigator.language || navigator.userLanguage).toLowerCase();
```

Comment:
Can't help wondering if we should leave this as a fallback (until we've done more cross-browser testing) to make sure this is always used as a last resort?